src/ride/track.h

```diff
@@ -292,6 +292,7 @@ enum {
 	TRACK_ELEM_BRAKES,
 	TRACK_ELEM_ROTATION_CONTROL_TOGGLE = 100,
 	TRACK_ELEM_INVERTED_90_DEG_UP_TO_FLAT_QUARTER_LOOP,
+	TRACK_ELEM_MAZE = 101,
```

Comment:
This will share same value with `TRACK_ELEM_INVERTED_90_DEG_UP_TO_FLAT_QUARTER_LOOP`, is that intended?